### ISSUE TITLE
ISBN didn't find books

### ISSUE THREAD

**Issue by ceccadroid** [anpigon/obsidian-book-search-plugin#49]
I'm  using Google as provider for my search but I didn't find some books neither with title nor with the ISBN (10 or 13) from Gogle Books or Goodreads.
An example is "Linger" from Maggie Stiefvater (english version in this case)
What can I do?
Thank in advance for any help

**Comment by fkuersch** [comment]
@ceccadroid Is your "Preferred locale" in the plugin settings set to english?

related #57

**Comment by ceccadroid** [comment]
@fkuersch Sorry, I completely miss the notification 😞 My locale is set to Italian since the majority of my book are in this language. May I need to swith the locale before searching for books in other languages?

**Comment by fkuersch** [comment]
Yes, indeed. The current implementation strictly filters by language, so search results in other languages are hidden (even if you enter the matching ISBN)

**Comment by rtm** [comment]
I'm unable to find this book: 978-0193518322

The New Oxford Organ Method
Anne Marsden Thomas and Frederick Stocken
Oxford University Press

This book refuses to come up whether searching by name, author or ISBN. I've tried the default locale and the English UK locale with no luck.

Any hints for how to debug this problem?
